ui/src/features/annotator/image-annotation.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Grid, View } from '@geti/ui';
+import { isEmpty } from 'lodash-es';
+
+import thumbnailUrl from '../../assets/mocked-project-thumbnail.png';
+import { ZoomProvider } from '../../components/zoom/zoom';
+import { ZoomTransform } from '../../components/zoom/zoom-transform';
+import { response } from '../dataset/mock-response';
+import { Annotations } from './annotations-canvas';
+
+type Item = (typeof response.items)[number];
+
+export const ImageAnnotations = ({ mediaItem, isFocussed }: { mediaItem: Item; isFocussed: boolean }) => {
+    const size = { width: mediaItem.width, height: mediaItem.height };
+
+    return (
+        <ZoomProvider>
```

Comment:
Note: this provider will need to be moved outside of this component if we want the primary toolbar and footers to make use of its values (e.g. to show zoom level and allow to zoom in / out)